src/api/__tests__/TaskCollaborators-test.js

```diff
@@ -5,6 +5,10 @@ let taskCollaborators;
 const BASE_URL = 'https://www.foo.com';
 const FILE_ID = 'foo';
 const USER_ID = 'userone';
+const headers = {
```

Comment:
Can these be imported from the class under test, instead, to avoid duplication in the test?